dbusmock/templates/bluez5.py

```diff
@@ -187,8 +187,8 @@ def AddDevice(self, adapter_device_name, device_address, alias):
 
 
 @dbus.service.method(BLUEZ_MOCK_IFACE,
-                     in_signature='ss', out_signature='')
-def PairDevice(self, adapter_device_name, device_address):
+                     in_signature='ssi', out_signature='')
+def PairDevice(self, adapter_device_name, device_address, class_=5898764):
```

Comment:
This is not actually a backwards-compatible change - D-Bus does not support default parameters. But `PairDevice()` is not an official API of bluez anyway (that would be [Pair()](https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/doc/device-api.txt)), and we don't even properly cover this in tests as it's too flaky. So I guess this is okay then.